cluster/src/main/java/com/hartwig/pipeline/tertiary/virusbreakend/VirusBreakendCommand.java

```diff
@@ -0,0 +1,28 @@
+package com.hartwig.pipeline.tertiary.virusbreakend;
+
+import com.hartwig.pipeline.calling.command.VersionedToolCommand;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.tools.Versions;
+import static com.hartwig.pipeline.resource.ResourceNames.VIRUSBREAKEND_DB;
+
+public class VirusBreakendCommand extends VersionedToolCommand {
+
+    public VirusBreakendCommand(ResourceFiles resourceFiles, String tumorSample, String tumorBamPath) {
+        super("gridss",
+                "virusbreakend.sh",
+                Versions.GRIDSS_VIRUS_BREAKEND,
+                "--output",
+                VmDirectories.OUTPUT + "/" + tumorSample + ".virusbreakend.vcf",
```

Comment:
There are some helper methods on `VmDirectories` to create paths.

Reply:
Ok will use those indeed.